### PR TITLE
Qt GUI: Windows Packaging

### PR DESCRIPTION
**Pull request by cjee21**
Qt GUI: Add files for MSIX packaging
- Needed files such as manifest and assets

Qt GUI: Add build and packaging script
- Script that generates an installable MSIX package of the Qt GUI for Windows from source files. Includes building, packaging and signing.

Features: The generated MSIX package should be installable on Windows 10 build 19000 and newer. The File Explorer context menu entry will appear shortly after installation and disappear immediately after uninstallation on Windows 10 and 11 (including modern menu). It can be enabled/disabled via the GUI Preferences and takes effect immediately. No registry entries will be written to the actual registry for the GUI settings as it uses registry virtualization which ensures a clean uninstallation. All features including Graph and FFmpeg plugins are available.

Note: Requires the IExplorerCommand shell extension project and source files for building the shell extension DLL which will be made available in a separate PR for Windows GUI: Windows 11 File Explorer Context Menu.

### CODE REVIEW

**Comment by cjee21** [comment]
The manifest has to be updated when there is a new version or if it is to be published to the Microsoft Store.

**Comment by JeromeMartinez** [comment]
Thank you. We merge that for the future.

> Windows 10 build 19000 and newer

It is currently a blocker for a release but it is fair enough right now for being included in the main source code.

File: Source/WindowsQtPackage/AppxManifest.xml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+<Package
+  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities"
+  xmlns:desktop="http://schemas.microsoft.com/appx/manifest/desktop/windows10"
+  xmlns:desktop4="http://schemas.microsoft.com/appx/manifest/desktop/windows10/4"
+  xmlns:desktop5="http://schemas.microsoft.com/appx/manifest/desktop/windows10/5"
+  xmlns:com="http://schemas.microsoft.com/appx/manifest/com/windows10"
+  IgnorableNamespaces="uap rescap desktop desktop4 desktop5 com">
+  <Identity Name="MediaInfo.Qt" Publisher="CN=MEDIAAREA.NET, O=MEDIAAREA.NET, L=Curienne, S=Auvergne-Rhône-Alpes, C=FR" Version="24.11.0.0" ProcessorArchitecture="x64" />
```
Comment (JeromeMartinez) [inline Source/WindowsQtPackage/AppxManifest.xml:11]: @g-maxime please update the version upgrade script for managing this new field.

Comment (cjee21) [reply Source/WindowsQtPackage/AppxManifest.xml:11]: This should be the only file remaining that is not yet in the upgrade script.

**Pull request merged**